pages/api/user/index.ts

```diff
@@ -19,7 +25,26 @@ export default async function handle(
             break;
         }
         case "POST": {
-            // TODO:
+            let newUser;
+            const userRole = req.body.role;
+            // TODO (6/27/21): define these roles as an enum somewhere? but where?
+            if (userRole === "PARENT") {
+                newUser = createParent(req.body, userRole);
```

Comment:
Not sure if `req.body` is actually what should be passed in - I was playing around a bit with postman and it looks like req.body is a string?

Reply:
What do you mean by "req.body is a string"?

Reply:
I mean `console.log("BODY:", typeof req.body);` prints `BODY: string`